src/components/HTMLEngineProvider/HTMLRenderers/AnchorRenderer.js

```diff
@@ -99,8 +99,15 @@ const AnchorRenderer = (props) => {
             key={props.key}
             displayName={displayName}
 
-            // Only pass the press handler for internal links, for public links fallback to default link handling
-            onPress={(internalNewExpensifyPath || internalExpensifyPath) ? navigateToLink : undefined}
+            // Only pass the press handler for internal links, for public links or whitelisted internal links fallback to default link handling
+            onPress={
+                (
+                    (internalNewExpensifyPath || internalExpensifyPath)
+                    && !CONST.PATHS_TO_TREAT_AS_EXTERNAL.includes(attrPath)
```

Comment:
Ah actually, I guess it is fine either way in this scenario. Since the path is just `NewExpensify.dmg`

Reply:
>It might be cleaner but this is the first place where we get the option to make the decision.

Sorry, I don't follow. Mind explaining this to me? Don't we define the `internalNewExpensifyPath` constant up on line 31? 

>If you think otherwise, I can change it though.

I'm not sure. Just looking to have a discussion to decide.